openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/v1/webhooks/VerificationRequestEvent.java

```diff
@@ -0,0 +1,232 @@
+/*
+ * Verification | Sinch
+ * Verify users with SMS, flash calls (missed calls), a regular call, or data verification.
+ *
+ * The version of the OpenAPI document: 2.0.1
+ * Contact: support@sinch.com
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+package com.sinch.sdk.domains.verification.models.v1.webhooks;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.sinch.sdk.core.utils.EnumDynamic;
+import com.sinch.sdk.core.utils.EnumSupportDynamic;
+import com.sinch.sdk.domains.verification.models.v1.Price;
+import com.sinch.sdk.domains.verification.models.v1.internal.IdentityInternal;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+/** declaration */
+@JsonDeserialize(builder = VerificationRequestEventImpl.Builder.class)
+public interface VerificationRequestEvent {
+
+  /**
+   * The ID of the verification request.
+   *
+   * @return id
+   */
+  String getId();
+
+  /** The type of the event. */
+  public class EventEnum extends EnumDynamic<String, EventEnum> {
+    public static final EventEnum VERIFICATIONREQUESTEVENT =
+        new EventEnum("VerificationRequestEvent");
+
+    private static final EnumSupportDynamic<String, EventEnum> ENUM_SUPPORT =
+        new EnumSupportDynamic<>(
+            EventEnum.class, EventEnum::new, Arrays.asList(VERIFICATIONREQUESTEVENT));
```

Comment:
Nit: when the enum contains a single value, it can use `Collections.singletonList()`